contracts/libraries/DnGmxJuniorVaultManager.sol

```diff
@@ -1561,6 +1793,10 @@ library DnGmxJuniorVaultManager {
         return abi.encodePacked(state.weth, uint24(500), state.usdc);
     }
 
+    function USDC_TO_WETH_(State storage state) internal view returns (bytes memory) {
```

Comment:
Here `USDC_TO_WBTC` and `USDC_TO_WETH` are giving incorrect paths. Hence, `USDC_TO_WETH_` was added to avoid any code break but we need to fix these two paths.